packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -172,6 +179,46 @@ export default class EditorUI {
 		return this._editableElementsMap;
 	}
 
+	/**
+	 * Returns viewport offsets object
+	 * { top: Number, right: Number, bottom: Number, left: Number }
+	 * Only top property is currently supported
```

Comment:
Missing:

*   A period at the end.
*   Blank link to separate paragraphs of text.